Assets/Scripts/Utilities/RenderSettingsManager.cs

```diff
@@ -64,18 +71,28 @@ void IChangeTargetRenderSettings( RenderSettingsData renderSettings )
 
 	void ApplyRenderSettings()
 	{
-		Color skyboxColor = Color.Lerp( _currentRenderSettings.dayColor,
-		                                _currentRenderSettings.nightColor,
+		Color skyboxColor = Color.Lerp( _currentRenderSettings.daySettings.color,
+		                                _currentRenderSettings.nightSettings.color,
 		                                timeOfDay );
 		RenderSettings.skybox.SetColor( "_Tint", skyboxColor );
```

Comment:
That's how you change the settings on a material. There's a way to do it with something called nameID. I'll have to figure that out at some point.